modules/serverless-rn-1-6-0.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-6-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.6.0
+// add a version, e.g. Technology Preview 1.0.0
+
+[id="new-features-1-6-0_{context}"]
+== New features
+* {ServerlessProductName} 1.6.0 is available on {product-title} 4.3 and newer versions.
+* {ServerlessProductName} has been updated to use Knative Serving 0.13.1.
+* {ServerlessProductName} has been updated to use Knative `kn` CLI 0.13.1.
+* {ServerlessProductName} has been updated to use Knative Serving Operator 0.13.1.
+* The `serving.knative.dev` API group has now been fully deprecated, and is replaced by the `operator.knative.dev` API group.
++
+You must complete the steps described in the {ServerlessProductName} 1.4.0 release notes, related to updating from the `serving.knative.dev` API group to the `operator.knative.dev` API group, before you can upgrade to the latest version of {ServerlessProductName}.
++
+[IMPORTANT]
+====
+This change causes unspecified commands, such as `oc get knativeserving`, to become unreliable and not always work correctly.
+
+After you have completed the upgrade to {ServerlessProductName} 1.6.0, you will need to remove the old CRD to fix this issue.
+You can remove the old CRD by entering the following command:
+----
+$ oc delete crd knativeservings.serving.knative.dev
```

Comment:
I would include this step in the update procedure.

Reply:
We have an additional issue to include this as part of the uninstall section that I will be working on today.